documents-services/src/main/java/org/exoplatform/documents/notification/utils/NotificationUtils.java

```diff
@@ -52,7 +62,7 @@ public static String getDocumentLink(Node node, SpaceService spaceService, Ident
     return stringBuilder.toString() ;
   }
 
-  public static String getSharedDocumentLink(String nodeUuid, SpaceService spaceService, String spacePrettyName) {
+  public static String getSharedDocumentLink(Node sharedNode, SpaceService spaceService, String spacePrettyName) throws RepositoryException {
```

Comment:
Could you check that this function is not used by other modules ?

Reply:
checked and used only in documents module